workflow/rocoto/gfs_tasks.py

```diff
@@ -588,6 +588,22 @@ def ocnanalprep(self):
         deps.append(rocoto.add_dependency(dep_dict))
         dependencies = rocoto.create_dependency(dep=deps)
 
+        resources = self.get_resource('prepoceanobs')
+        task = create_wf_task('prepoceanobs',
```

Comment:
This method is no longer available.

Reply:
yes.  for example, please take a look at the `ocnanalprep` task below that has been updated.

Reply:
Corrected, with a couple of bugfixes

Reply:
PS also tested with a full cycle of WCDA